tests/unit/helpers/users.test.js

```diff
@@ -319,7 +319,7 @@ describe('getLearnerList', () => {
   });
 
   it('should get learners from company', async () => {
```

Comment:
Pour differencier des cas suivants, tu pourrais mettre ici "should get learners from company to date`

Reply:
done